practices/software_development_kit.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: software-development-kit
+stages:
+  - development
+short_description: A SDK is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so.
+tags:
+  - sdk
+  - software development
+  - software engineering
+  - development
+  - developer experience
+keywords:
+  - product sprint
+  - software development
+  - software engineering
+  - product development
+  - debugging
+  - documentation
+  - libraries
+  - apis
+  - sdk
+---
+
+# Software Development Kit (SDK)
+
+**TL;DR**
+
+It is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so. You have what you need to get the job done, and what's included in your kit toolbox, varies from platform to platform.
+
+## What is an SDK
+
+A Software Development Kit (SDK) is the toolbox of the software development world. But instead of tools like screwdrivers and pliers, SDKs contain libraries and Application Programming Interfaces (APIs) that make it easier to create applications for specific software and/or hardware.
+
+It can also be thought of as a wrapper library developed by a service provider which provides an abstraction around the services or capabilites that the platform has or supports.
+
+![SDK](/files/software_development_kit.png)
+[CleverTap: What is an SDK? Everything You Need to Know](https://clevertap.com/blog/what-is-an-sdk/)
+
+### Why You Might Want an SDK
+
+An SDK can include a range of things, including libraries, documentation, code samples, processes, and guides that developers can use and integrate into their applicatios. SDKs are designed to be used for specific platforms or programming languages.
```

Comment:
I can find similar text on https://clevertap.com/blog/what-is-an-sdk/ (which is not in the sources). Please, rewrite this in your own words. 

_SDK tools will include a range of things, including libraries, documentation, code samples, processes, and guides that developers can use and integrate into their own apps. SDKs are designed to be used for specific platforms or programming languages._